Traceback (most recent call last):
  File "/home/koji0708/Dropbox/01_projects/2018/competitive_programming/work/codeforces/466/A/A.py", line 66, in <module>
    solve()
  File "/home/koji0708/Dropbox/01_projects/2018/competitive_programming/work/codeforces/466/A/A.py", line 49, in solve
    N, d = map(int, input().split())
ValueError: not enough values to unpack (expected 2, got 1)
